playbooks/ansible-cloud/vcsa-appliance/files/bootstrap.yaml

```diff
@@ -1,4 +1,52 @@
 ---
+- hosts: controller
+  tasks:
+    - name: Add a vcenter entry in /etc/hosts
```

Comment:
we can move this into files/bootstrap.yaml, which will allow us to abstract this playbook out into shared content between network /cloud jobs.